iml-api/src/graphql/mod.rs

```diff
@@ -1139,6 +1027,32 @@ impl MutationRoot {
         transaction.commit().await?;
         Ok(true)
     }
+
+    #[graphql(arguments(
+        filesystem(description = "The filesystem to remove snapshot policies for"),
+        id(description = "Id of the policy to remove"),
+    ))]
+    /// Removes the automatic snapshot policy.
+    async fn remove_snapshot_policy(
+        context: &Context,
+        filesystem: Option<String>,
+        id: Option<i32>,
+    ) -> juniper::FieldResult<bool> {
+        sqlx::query!(
+            r#"
+                DELETE FROM snapshot_policy
+                WHERE (filesystem IS NOT DISTINCT FROM $1)
+                OR    (id IS NOT DISTINCT FROM $2)
```

Comment:
Why not !=

Reply:
Likewise, `$2` can be NULL. I don't actually use `id`. So here it's just a PoC.

Reply:
Yeah, I think we should eliminate the id and just make the fs field required.

Reply:
... and it will be `=` ;-)